tests/test_bootstrap.py

```diff
@@ -400,8 +400,8 @@ def mock_ensure_config_exists():
     log_no_color = Mock()
 
     with patch(
-        "homeassistant.config.async_hass_config_yaml", return_value={"browser": {}}
-    ):
+        "homeassistant.config.async_hass_config_yaml", return_value={"map": {}}
+    ), patch("homeassistant.components.person.async_setup", side_effect=[False, True]):
```

Comment:
What's the second dependency that needs this?

Reply:
Well that's why I didn't initially check if frontend wanted to be loaded because it's a dependency of a lot of built-in integrations that register panels, like logbook etc . In this case it's map.

Reply:
And the frontend relies on like 6 different integrations.

Reply:
I was wondering specifically about the list we return as side effect. The first time the side effect will be False and the second time True. I was thinking the first time person is set up it's a dependency of map. What's the reason the second time should return True?

Reply:
Home Assistant caches setup results. But when safe mode kicks in after frontend failed to set up, we're creating a new hass object to set up safe mode. So the new hass instance will need to have have `person` configured correctly. I guess a better test would be to misconfigure `person` to begin with, I'll update the test.

Reply:
Updated. Now the test will also verify that we don't use the users config anymore.